docs/can-guides/commitment/recipes/playlist-editor/playlist-editor.md

```diff
@@ -765,6 +765,6 @@ Update the `JS` tab to:
 @sourceref ./8-create-playlist.js
 @highlight 12-19,99-141,only
 
-Congrats! You've completed this guide.
+Congrats! If you've completed this recipe, you now have your very own YouTube Playlist Editor.
```

Comment:
I think the optimism of the original text is more encouraging. Maybe _Congrats! You now have your very own YouTube Playlist Editor._